### DISCUSSION TITLE
Pymakr 2.0 extension missing progress bar

### DISCUSSION THREAD

**Discussion by MBakerParagon** [pycom/pymakr-vsc#234] [General]
Perhaps I am missing something obvious here, but where is the indication for syncing the project to the device. I can see there is a busy hover over indicator, but no way to tell when it is done. This is problematic for me since I have a project with a large number of files. In the previous extension it would list the file name and size that it was syncing. Did that move or was that removed?

Am I missing something obvious here?

**Comment by jakobrosenberg** [comment]
@MBakerParagon thanks for bringing this up. I'll see if I can find a solution to this.

**Comment by jakobrosenberg** [comment]
This has been fixed in https://github.com/pycom/pymakr-vsc/commit/aacee2d3f6dc7f9edb4a8fc09faeaa9f407344b3 and is currently available in the [preview version](https://marketplace.visualstudio.com/items?itemName=pycom.pymakr-preview).

![upload](https://user-images.githubusercontent.com/4153004/170029390-cdbd1a00-7d12-44b1-9eeb-f626d2a900e9.gif)

I hope this fixes your issue. Sorry about the late reply, but I don't check the discussions as frequent as the issues. 🙂

EDIT: Fixed bad link for [preview version](https://marketplace.visualstudio.com/items?itemName=pycom.pymakr-preview)
